### PR TITLE
chore: Dockerize cardano-services http-server, extend docker-compose services

### PR DESCRIPTION
**Pull request by rhyslbw**
# Context
We're missing a Dockerfile and Docker Compose stack to run the `cardano-services` service dependencies.

# Proposed Solution
- Add a Docker Compose file with service dependencies defined
  
# Important Changes Introduced
- The docker-compose file is now in the `cardano-services` package scope.
- `run.ts` now supports loading secrets to conform with existing patterns involving `postgres` and `cardano-db-sync` Docker services.

### CODE REVIEW

**Review by ssong-iohk — CHANGES_REQUESTED** [review]

File: packages/cardano-services/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      - ${OGMIOS_PORT:-1337}:1337
+    restart: on-failure
+    volumes:
+      - node-db:/db
+      - node-ipc:/ipc
+
+  cardano-db-sync-extended:
+    image: inputoutput/cardano-db-sync:${CARDANO_DB_SYNC_VERSION:-12.0.0}
+    command: [
+        "--config", "/config/cardano-db-sync/config.json",
```
[the remark this answers is not in the document]
Comment (rhyslbw) [reply packages/cardano-services/docker-compose.yml:42]: It must be an issue with the config mount, as this path is correct. See the [volume mapping](https://github.com/input-output-hk/cardano-js-sdk/pull/294/files#diff-a5f22c45d19dce606e9bae0a8cd3a11ce586442845b16c7e1a056ac7e028ea79R59) into the root

Maybe you don't have the submodule?

[the remark this answers is not in the document]
Comment (ssong-iohk) [reply packages/cardano-services/docker-compose.yml:42]: Ah. you right. I updated `git submodule` after seeing the error. my bad!

**Review by ssong-iohk — APPROVED** [review]
Thanks @rhyslbw it's handy one.

**Review by mkazlauskas — DISMISSED** [review]
LGTM! :rocket:

**Review by AngelCastilloB — APPROVED** [review]
Looks good :)

**Pull request merged**